content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst

```diff
@@ -25,81 +31,74 @@ depends on the stage of the manufacturing process.
    :guilabel:`Search...` bar to display all virtual locations. From the list, select the
    :guilabel:`Virtual Locations/Scrap` location.
 
-Scrap manufacturing components
-==============================
+.. _scrap_components/scrap-window:
 
-To scrap components during the manufacturing process, begin by navigating to
-:menuselection:`Manufacturing --> Manufacturing Orders`, then select a manufacturing order or click
-:guilabel:`Create` to configure a new one. If a new manufacturing order is created, select a product
-from the :guilabel:`Product` drop-down menu, then click :guilabel:`Confirm`.
+Scrap pop-up window
+===================
 
-Once the manufacturing order has been confirmed, a :guilabel:`Scrap` button appears at the top of
-the page. Click the button and a :guilabel:`Scrap` pop-up window appears.
+Scrapping components and finished products is done through the :guilabel:`Scrap` pop-up window.
+The pop-up window can be accessed from an |MO| in the backend, or the Shop Floor module.
```

Comment:
```suggestion
The pop-up window can be accessed from an |MO| in the backend, or the *Shop Floor* module.
```
just adding the necessary * to the module name here. Like we discussed, you don't have to add the 'Scrap' title to this sentence re: pop-up window, as it was just mentioned the sentence prior -- as you pointed out yesterday 👍